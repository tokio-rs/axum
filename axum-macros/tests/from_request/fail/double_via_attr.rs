use axum_macros::FromRequest;

#[derive(FromRequest)]
struct Extractor(#[from_request(via(axum::Extension), via(axum::Extension))] State);

#[derive(Clone)]
struct State;

fn main() {}
