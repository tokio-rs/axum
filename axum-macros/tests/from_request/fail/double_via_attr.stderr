error: `via` specified more than once
 --> tests/from_request/fail/double_via_attr.rs:4:55
  |
4 | struct Extractor(#[from_request(via(axum::Extension), via(axum::Extension))] State);
  |                                                       ^^^
